pom.xml

```diff
@@ -60,6 +61,12 @@
             <version>4.11</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>${commons-io.commons-io.version}</version>
```

Comment:
For test-scope dependencies, we should just specify the version number inline to avoid confusion to readers of the POM's properties.